<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Changing Colorspaces</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApplyMode">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonImage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonWebcam">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Webcam</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserChooseFile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChooseFile">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonVideo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
